roles/internal/grok/tasks/install.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+- name: Install Grok dependencies
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - cmake
+    - libpng16-dev
+    - libtiff-dev
+    - liblcms2-dev
+
+- name: Clone Grok
+  git:
+    repo: https://github.com/GrokImageCompression/grok.git
+    dest: "{{ grok_clone_directory }}"
+    version: "{{ grok_version_tag }}"
+
+- name: Make build directory
+  file:
+    path: "{{ grok_clone_directory }}/build"
+    state: directory
+    mode: 0755
+
+- name: Cmake grok
+  command: cmake ../
+  args:
+    chdir: "{{ grok_clone_directory }}/build"
+    creates: "{{ grok_clone_directory }}/build/*.cmake"
+      
+- name: Make grok
+  command: make
+  args:
+    chdir: "{{ grok_clone_directory }}/build"
+    creates: "{{ grok_clone_directory }}/build/bin/opj_decompress"
+
+- name: Install grok
+  command: make install
+  args:
+    chdir: "{{ grok_clone_directory }}/build"
+    creates: /usr/local/bin/opj_decompress
+
+- name: Update ldconfig
```

Comment:
So to get rid of the last idempotency thing, I'd do something like this:

```yaml
- name: Install grok
  command: make install
  args:
    chdir: "{{ grok_clone_directory }}/build"
    creates: /usr/local/bin/opj_decompress
    register: grok_make_install

- name: Update ldconfig
  command: ldconfig
  when: grok_make_install.changed is defined AND grok_make_install.changed
```

Reply:
Your `when:` line above caused an error when I tried to run it. I was able (I think) to get the expected process with `when: grok_make_install.changed`. Does that seem viable?